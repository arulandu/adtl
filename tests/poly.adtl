id := fun A : Type 0 => fun x : A => x.
Context.