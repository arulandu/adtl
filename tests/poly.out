id = fun A : Type 0 => fun x : A => x
    : forall A : Type 0, A -> A
